lib/Advisor.js

```diff
@@ -27,7 +27,6 @@ class Advisor {
     this.panel = this.createPanel();
     this.createRequestListeners();
     this.createWindowListeners();
-    this.handleSyncChange();
```

Comment:
Wouldn't you still want to check whether the user has sync on startup?

Although if you did keep this function here it might be nice to name it something more accurate about what it's doing, since it would not longer be only used as an event handler.